src/Tribe/PUE/Notices.php

```diff
@@ -95,15 +94,26 @@ protected function populate() {
 			return;
 		}
 
+		/**
+		 * @todo - redscar
+		 * Potentially change with -
+		 * wp_parse_args or array_merge
+		 */
```

Comment:
Not potentially this needs to happen ! and actually ive looked the code i dont get why this needs to be a merge at all....

why not $this->notices = $this->saved_notices ?